eclass/qt5-build.eclass

```diff
@@ -675,6 +675,9 @@ qt5_base_configure() {
 		# do not build with -Werror
 		-no-warnings-are-errors
 
+		# select profile based on current compiler
+		-platform $(qt5_get_mkspec)
```

Comment:
we don't want to affect existing stable ebuilds, so this needs to be conditional, let's say 5.10 and later